src/include/gen/Firebird.pas

```diff
@@ -15196,6 +15197,7 @@ function ITraceProcedureImpl_getPerfDispatcher(this: ITraceProcedure): Performan
 
 function ITraceProcedureImpl_getStmtIDDispatcher(this: ITraceProcedure): Int64; cdecl;
 begin
+	Result := 0;
```

Comment:
Interesting and sooner of all useful fix but:
- unrelated with particular PR
- should be accomplished with changes in cloop sources